### ISSUE TITLE
Details of pretraining

### ISSUE THREAD

**Issue by LoveMiki** [WentaoChen0813/SemanticPrompt#3]
Hi

I notice that the scheduler is set as None by default and the scripts do not include the parameter of the scheduler.
Did you use cosine annealing in the pre-training?

Thanks
